### ISSUE TITLE
Install Issues & Service Issues - Need More Development Please....

### ISSUE THREAD

**Issue by sla0101** [nefarius/ScpToolkit#1026]
Hello,

Is the development still active on this project?

I'm running Win7 SP1 x64 Home edition.

First problem, I got a popup saying disk1.cab had invalid digital signature.

Then I installed the Window Service, and I received a popup on the lower right of the desktop, saying it's already installed, like hmm, I didn't need this?

Then the SCP Dsx Service wouldn't start, something to do with a dependency issue, sorry I forget the exact wording.

Bottom line, I hope there will be some active development soon to get this up and working betting in Windows 7...

Thanks

**Comment by sla0101** [comment]
Sorry my mistake, I didn't try hard enough, everything is working! :)

But I do hope that there will be more development in the near future.

Like there is no GUI to reassign the controller layout if someone wanted? 

Thanks

**Comment by K9998f** [comment]
hi bro do you know how to change the quic disconnect combo?
